### PR TITLE
release: v0.7.1

### PR DESCRIPTION
**Pull request by jma**
* Fixes errors for production build due to the @ngneat/hotkeys library.

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on `rero-ils#<xx>`'s PR(s):

* rero/rero-ils#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

**Review by iGormilhit — APPROVED** [review]

**Review by zannkukai — COMMENTED** [review]
approved even after merging !

**Pull request merged**
